router/openwrt/setup.go

```diff
@@ -122,6 +155,14 @@ func (r *Router) Restore() error {
 	// Remove the custom dnsmasq config
 	_ = os.Remove(r.DNSMasqPath)
 
+	// Remove the DHCP option 6 (DNS server) if it was set
+	if _, err := uci("del", "dhcp.lan.dhcp_option"); err != nil {
```

Comment:
Is there a chance this option was available before? Could removing it leave the router misconfigured?

Reply:
Good point, we can use `uci del_list dhcp.lan.dhcp_option="6,router_ip_here"` to only remove the added configuration. I have updated the PR. `del_list` doesn't throw an error when `dhcp_option` is empty or if the specified option is not included in the current configuration.